cmake/modules/CheckYasm.cmake

```diff
@@ -22,21 +22,21 @@ macro(check_yasm_support _object_format _support_x64 _better_result)
           ${CMAKE_SOURCE_DIR}/src/isa-l/include/
           ${CMAKE_SOURCE_DIR}/src/isa-l/erasure_code/gf_vect_dot_prod_avx2.asm
           -o /dev/null
-          RESULT_VARIABLE rc
+          RESULT_VARIABLE not_support_avx2
           OUTPUT_QUIET
           ERROR_QUIET)
-        if(NOT rc)
-          set(${_better_result} TRUE)
-        endif(NOT rc)
+        if(NOT not_support_avx2)
+          set(${_support_x64_and_avx2} TRUE)
+        endif(NOT not_support_avx2)
       endif(not_arch_x32)
     endif(CMAKE_SYSTEM_PROCESSOR MATCHES "amd64|x86_64")
   endif(NOT no_yasm)
   if(no_yasm)
     message(STATUS "Could NOT find Yasm")
   elseif(NOT not_arch_x32)
-    message(STATUS "Found Yasm: but x86_64 with x32 ABI is not supported")
-  elseif(${_better_result})
-    message(STATUS "Found Yasm: good -- capable of assembling x86_64")
+      message(STATUS "Found Yasm: only support ILP32 i.e. x86_32")
```

Comment:
why reindent?

Reply:
ILP32 is how we detect the x32 ABI. and there is no such a thing like x86_32. also why s/x32/x86_32/ ? x32 is the name of this ABI. it is a well known name. i am not sure about x86_32. it's is not officially defined anywhere.

Reply:
@tchaikov Thanks for your info. I agree with you. I'll keep the original message there.